src/redux/sagas/application-controller.ts

```diff
@@ -52,10 +53,19 @@ export default function* applicationControllerSaga(wallet: ChannelWallet) {
 
     if (newState) {
       if (newState.isReadyToSend) {
+
         yield put(MessageAction.sendMessage(newState.opponentAddress, newState.move.toHex()))
         yield put(GameAction.moveSent(newState.move));
       }
-      yield put(GameAction.stateChanged(newState));
+      // Fake sending a transaction for now
+      if (newState instanceof ReadyToDeploy && gameEngine){
+          gameEngine!.transactionSent();
```

Comment:
Just a quick cheat to avoid another null check. `gameEngine!.transactionSent()` lets the compiler know that gameEngine will not be null, so it won't complain about it. Since this is just faking a transaction for now I figured it would be ok.